src/hooks/sync-sites/use-sync-push.ts

```diff
@@ -99,7 +99,7 @@ export function useSyncPush( {
 						'The site is too large to push. Please reduce the size of the site and try again.'
 					),
 				} );
-				return;
+				return false;
```

Comment:
Instead of returning true and false, we can call `await refreshTooltip( selectedSite.id, connectedSite.id, 'push' );` in the success case inside this hook.
Or maybe these hooks can accept a onSuccess callback.